### ISSUE TITLE
Pretraining of the generator and discriminator SIFA

### ISSUE THREAD

**Issue by aeijpe** [abc-xyz1/ADR#7]
Hii, thanks for the nice and interesting work and code! 

I have a question on the pretrained weights of the Generator and Discriminator from SIFA that you are using.
In the code you directly obtained the pretrained weights from SIFA in the readCkpt.py. In the paper, you mention that “For the generator Gt and target domain discriminator Dt in coarse alignment, we load the parameters pretrained by the cycle consistency loss from SIFA”. 

My question here is; did you run the whole SIFA model on your data (or just took the weights directly as you are using the same dataset)? Or are you retraining an adapted version of the SIFA training procedure with only the cycle consistency loss? I am asking because in the code for training SIFA, the generator and discriminator are indeed optimized with the cycle images (and cycle consistency loss). However, these obtained cycle images also pass through the encoder, which is (amongst others)  optimized by using the labels from the images. Therefore to optimize the generator and discriminator, you (indirectly) also need the other losses and image labels. Thus, I am wondering if you use the whole, original SIFA training code or really only the cycle consistency loss with an adapted version of the SIFA training code and if so, would it be possible to push that adapted code also to GitHub? 

I am looking forward to your response!
